### DISCUSSION TITLE
One package exceeds maximum ececution time.

### DISCUSSION THREAD

**Discussion by AndiLeni** [composer/composer#9985] [Q&A / Support]
Hello, 
I have problems with an application in my CMS (Redaxo).
For this I install the package matomo/device-detector, which in turn requires https://github.com/mustangostang/spyc.
Exactly here now a problem occurs.
My CMS has its own autoloader which works automatically.
If I call the autoload.php of composer with require_once, the system enters an endless loop and throws an error because the max_execution time is exceeded.
I was able to identify the spyc package as the problem, as it seems to be the only one that is loaded as a "file"? or at least differently from the other packages as fas as I can see.

What could be the reason for this?
Or, can I tell composer not to load this one package at all?

Kind regards
Andreas

**Comment by Seldaek** [comment]
There is no way to not load the package if it is installed. I don't really see how including Spyc.php would trigger a max execution time though..
